apps/system/js/notifications.js

```diff
@@ -122,6 +126,88 @@ var NotificationScreen = {
     }, function(err) {
       console.error('VersionHelper failed to lookup version settings.');
     });
+
+
+  },
+
+  getStore: function ns_getStore() {
+    return new Promise((resolve) => {
+      if (this._storeRef) {
+        return resolve(this._storeRef);
+      }
+      navigator.getDataStores(this.STORE_NAME).then((stores) => {
+        this._storeRef = stores[0];
+        return resolve(this._storeRef);
+      }, (reason) => {
+        return reject(reason);
+      });
+    });
+  },
+
+  increaseAppCounter: function ns_udpateCounter(detail) {
+    this.debug('increaseAppCounter('+JSON.stringify(detail)+')');
+    this.debug('has url = ' + this._countersPerApp.hasOwnProperty(detail.manifestURL));
```

Comment:
Could you remove debug information from system application that connected to homescreen notification.